Corona-Warn-App/src/test/java/de/rki/coronawarnapp/datadonation/analytics/common/PpaDataExtensionsTest.kt

```diff
@@ -26,4 +30,69 @@ class PpaDataExtensionsTest : BaseTest() {
         PpaData.PPAFederalState.FEDERAL_STATE_SH.federalStateShortName shouldBe "SH"
         PpaData.PPAFederalState.FEDERAL_STATE_TH.federalStateShortName shouldBe "TH"
     }
+
+    @Test
+    fun `days since most recent date at risk level at test registration are calculated correctly`() {
+        val now = Instant.now()
+        val dayBeforeYesterdayAt2200 = now
```

Comment:
Maybe use `Instant.parse("2021-03-15T22:00:00.000Z")` in these tests here to simplify creation of instants?